keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resolvers/ImageResolver.kt

```diff
@@ -146,7 +160,7 @@ class ImageResolver(
       spec = spec.copy(
         overrides = overrides,
         _artifactName = image.artifact?.name
-          ?: error("Artifact not found in image ${image.namedImage.imageName}"),
+          ?: error("Artifact not found in images ${image.namedImages.values.map { it.imageName }}"),
```

Comment:
Interestingly the Jenkins image path always sets `image.artifact` to `null` which makes me believe it flat out doesn't work any more. Planning a further PR to remove the deprecated image providers